bitvision/bitvision.py

```diff
@@ -15,126 +15,160 @@
 CONFIG = configparser.ConfigParser()
 CONFIG.read("config.ini")
 
-
 #########
 # HELPERS
 #########
 
+# BitVision Command Line Trading Platform
+# Aaron Lichtman and Jon Shobrook
 
-def login():
+import os
+import sys
+import urwid
+
+# Place Your Own Trade is not shown until Trading Deactivated.
+choices = ["Activate/Deactivate Automated Trading", "Deposit/Withdraw Money", "Place Your Own Trade"]
+
+
+def menu(title, choices):
```

Comment:
This function needs to be in the `App` object defined below. When  `App` is initialized, it gathers all the needed data, loads them into widgets, and then displays them. `App` will also have functions for autorefreshing the CLI and handling input.

Reply:
I removed this block of code in favor of the footer menu bar.